### ISSUE TITLE
Incompatible with eslint >= 5.2.0

### ISSUE THREAD

**Issue by czycha** [zouhir/eslint-config-standard-preact#11]
When installing this package with eslint 5.2.0 or 5.3.0 already installed, I receive the following error:

```
An unexpected error occurred: "could not find a copy of eslint to link in /Users/james/Desktop/Work/Timberwolves/enter-to-win/node_modules/eslint-config-standard-preact/node_modules".
```

I have to have eslint 5.1.0 installed to install this package.

**Comment by tardnuk** [comment]
Do you use yarn? When I define exact version in package.json, it's fine - "eslint": "5.7.0".

**Comment by czycha** [comment]
@tardnuk I do use Yarn. You're right, installing the exact version works even though it's the same. I had thought that perhaps this module was not configured correctly but now I think this is a bug with Yarn. Some other people have reported this with other modules: https://github.com/yarnpkg/yarn/issues/6285. 

I'm closing this issue but feel free to reopen if there's anything to be done here to fix this issue.
